.github/release-drafter.yml

```
@@ -26,3 +26,25 @@ categories:
       - 'test flakiness'
   - title: 📦 Dependency updates
     label: 'dependencies'
+autolabeler:
+  - label: 'breaking change'
+    title:
+      - '/^break(\(.+\))?\:/i'
+  - label: 'security'
+    title:
+      - '/^security(\(.+\))?\:/i'
```

bug: because of the above all other patterns need to be updated, for example:
```suggestion
      - '/^security(\(.+\))?!?\:/i'
```